src/redux/books/books.js

```diff
@@ -1,25 +1,27 @@
-const INCREMENT = 'react-redux-setup/counter/increment';
-const DECREMENT = 'react-redux-setup/counter/decrement';
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
 
 const initialState = {
-  count: 0,
+  books: [],
 };
 
 export default (state = initialState, action) => {
   switch (action.type) {
-    case INCREMENT:
-      return { ...state, count: state.count + 1 };
-    case DECREMENT:
-      return { ...state, count: state.count - 1 };
+    case ADD_BOOK:
+      return { ...state, books: [...state.books, action.payload] };
+    case REMOVE_BOOK:
+      return { ...state, books: state.books.filter((book) => book.id !== action.payload) };
```

Comment:
- [ ] [OPTIONAL] The first `...state` is not necessary, this works perfectly fine:
    ```javascript
      case ADD_BOOK:
      return { books: [...state.books, action.payload] };
      case REMOVE_BOOK:
      return { books: state.books.filter((book) => book.id !== action.payload) };
    ```